src/main/java/frc/team4909/robot/subsystems/elevator/ElevatorSubsystem.java

```diff
@@ -15,26 +15,32 @@
 import frc.team4909.robot.subsystems.elevator.commands.SetElevatorPosition;
 import frc.team4909.robot.Robot;
 import frc.team4909.robot.RobotConstants;
+import frc.team4909.robot.RobotMap;
 import frc.team4909.robot.operator.controllers.BionicF310;
 
 public class ElevatorSubsystem extends Subsystem{
-    WPI_VictorSPX VictorSPX1, VictorSPX2, VictorSPX3;
-    WPI_TalonSRX masterSRX;
+    WPI_VictorSPX leftSPX, rightSRX1, rightSRX2;
+    WPI_TalonSRX leftSRX;
 
     public int holdingPosition;
 
-    public void ElevatorSubsytem(){
+    public ElevatorSubsystem() {
 
         //Lift
-        masterSRX = new WPI_TalonSRX(RobotConstants.elevatorSRXID);  //master SRX
-        VictorSPX1 = new WPI_VictorSPX(RobotConstants.elevatorSPX1ID); //slave SPX 1
-        VictorSPX2 = new WPI_VictorSPX(RobotConstants.elevatorSPX2ID); //slave SPX 2
-        VictorSPX3 = new WPI_VictorSPX(RobotConstants.elevatorSPX3ID); //slave SPX 3
+        leftSRX = new WPI_TalonSRX(RobotMap.elevatorSRXID);  //master SRX
+        leftSPX = new WPI_VictorSPX(RobotMap.elevatorSPX1ID); //slave SPX 1
+        rightSRX1 = new WPI_VictorSPX(RobotMap.elevatorSPX2ID); //slave SPX 2
+        rightSRX2 = new WPI_VictorSPX(RobotMap.elevatorSPX3ID); //slave SPX 3
```

Comment:
These two "right" variables appear to be named incorrectly. The controller is SPX but the variable says it's SRX. I think these variable names should be `rightSPX1` and `rightSPX2`

Reply:
yep, good catch, will fix

Reply:
Fixed